WordPress/src/main/java/org/wordpress/android/ui/ActivityLauncher.java

```diff
@@ -82,6 +82,7 @@ public static void viewBlogPlans(Context context, int blogLocalTableId) {
         Intent intent = new Intent(context, PlansActivity.class);
         intent.putExtra(PlansActivity.ARG_LOCAL_TABLE_BLOG_ID, blogLocalTableId);
         slideInFromRight(context, intent);
+        AnalyticsUtils.trackWithCurrentBlogDetails(AnalyticsTracker.Stat.OPENED_PLANS);
```

Comment:
Using `trackWithCurrentBlogDetails` seems problematic here since this routine accepts a `blogLocalTableId` parameter. This works fine right now since it's always the ID of the current blog, but this could change in the future, and this is the sort of thing that would fall through the cracks.

I think we need a `trackWithBlogDetails` that accepts a blogId in order to resolve this.